src/lesson/lesson03.js

```diff
@@ -1174,51 +1271,48 @@ function Lesson03() {
       ),
       "",  // initialCode
       Constants.Lesson03.SUCCESS_MESSAGE,
-      20
+      25
```

Comment:
With this limit, this level allows a solution that does not use conditionals:
FFRGFPRRFRFFF 3{FRGFPRRFR}

Was that intentional? I don't think it's necessarily a problem, though.